packages/web/src/lib/parsing.ts

```diff
@@ -95,3 +95,23 @@ export function parseAppArgs(appArgs?: Array<Uint8Array | string>): Uint8Array[]
 	}
 	return args as Uint8Array[];
 }
+
+/**
+ * @param str string to convert
+ * @returns fooBar to foo-bar
+ */
+export function convertCapitalToHyphens(str: string): string {
+	return str.replace(/[A-Z]/g, (m) => "-" + m.toLowerCase());
+}
+
+/**
+ * @param object
+ * @returns object keys from fooBar to foo-bar
+ */
+export function convertKeysToHyphens(object: any) {
+	let newObject: any = {};
+	Object.keys(object).forEach((key) => {
+		newObject[convertCapitalToHyphens(key)] = object[key];
+	});
+	return newObject;
+}
```

Comment:
You have the same functions in runtime. Please implement at a single place. I think runtime can import from web.

Reply:
okay